src/dlgPackageExporter.cpp

```diff
@@ -466,12 +468,15 @@ void dlgPackageExporter::recurseKeys(TKey* item, QTreeWidgetItem* qItem)
 void dlgPackageExporter::listKeys()
 {
     KeyUnit* tu = mpHost->getKeyUnit();
-    list<TKey*>::const_iterator it;
-    std::list<TKey*> tList = tu->getKeyRootNodeList();
-    QList<QTreeWidgetItem*> items = treeWidget->findItems(QString("Keys"), Qt::MatchExactly, 0);
-    QTreeWidgetItem* top = items.first();
-    for (it = tList.begin(); it != tList.end(); it++) {
-        TKey* pChild = *it;
+    list<TKey *>::const_iterator it;
```

Comment:
The first six of these nine lines of change seem entirely down to white space differences where the new one does not seem to match our code style - they ought to be restored either by hand or using the Clang-format beautifier on this chunk of code IMHO!